ExampleService/ExampleService_WebApi/Controllers/TaskController.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExampleService_WebApi;
+
+[ApiController]
+[Route("[controller]")]
+public class TaskController :  ControllerBase
+{   
+    private readonly InterfaceClass _interface;
+
+    //injecting the interface dependency
+    public TaskController(InterfaceClass interfaceClass)
+    {
+        _interface= interfaceClass;
+    }
+
+    //to return the list of task
+    [HttpGet("List of your Task")]
+    public IActionResult Get()
+    {   
+        var taskList = _interface.ListofTask();
+
+        //mapping as DTO
+        List<TaskDTO> taskMapping = new List<TaskDTO>();        
```

Comment:
```suggestion
        List<TaskDTO> taskResponse= new List<TaskDTO>();        
```

Name the variable appropriately, Variable name itself should tell what it is used for without looking the code to understand what it is used for.

Reply:
Named variables according to the purpose.